spec/features/add_card_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+context "fill fields in form"  do
+    it 'fill fields in form' do
+      visit new_card_path
+        fill_in 'card[original_text]', with: 'word'
+        fill_in 'card[translated_text]', with: 'slovo'
+      click_button 'Create Card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.